ui/src/app/pages/analysis/notebook-redirect.spec.tsx

```diff
@@ -204,69 +203,4 @@ describe('NotebookRedirect', () => {
     expect(currentCardText(wrapper))
       .toContain(progressStrings.get(Progress.Redirecting));
   });
-
```

Comment:
IIRC these were covering specific regressions; is it not possible to make the same navigate assertions with memory router?